### ISSUE TITLE
Discovergy API changed

### ISSUE THREAD

**Issue by barni** [iobroker-community-adapters/ioBroker.discovergy#190]
Hi,
the Disvoergy API changed, meteringLocationNumber has been added to DiscovergyMeter.

Cheers,
Stephan

021-03-08 21:59:45.534	info	(69134) cleaned everything up...
discovergy.0	2021-03-08 21:59:45.510	error	(69134) The state contains the forbidden properties street, streetNumber, zip, city, country!
discovergy.0	2021-03-08 21:59:45.510	error	at IncomingMessage.emit (events.js:326:22)
discovergy.0	2021-03-08 21:59:45.510	error	at Object.onceWrapper (events.js:420:28)
discovergy.0	2021-03-08 21:59:45.510	error	at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/request/request.js:1076:12)
discovergy.0	2021-03-08 21:59:45.510	error	at Request.emit (events.js:314:20)
discovergy.0	2021-03-08 21:59:45.510	error	at Request.<anonymous> (/opt/iobroker/node_modules/request/request.js:1154:10)
discovergy.0	2021-03-08 21:59:45.510	error	at Request.emit (events.js:314:20)
discovergy.0	2021-03-08 21:59:45.510	error	at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
discovergy.0	2021-03-08 21:59:45.510	error	at Request._callback (/opt/iobroker/node_modules/iobroker.discovergy/main.js:90:13)
discovergy.0	2021-03-08 21:59:45.510	error	at Discovergy.setState (/opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:5935:21)
discovergy.0	2021-03-08 21:59:45.510	error	at validateSetStateObjectArgument (/opt/iobroker/node_modules/iobroker.js-controller/lib/adapter.js:5848:23)
discovergy.0	2021-03-08 21:59:45.510	error	(69134) Error: The state contains the forbidden properties street, streetNumber, zip, city, country!
discovergy.0	2021-03-08 21:59:45.508	error	(69134) unhandled promise rejection: The state contains the forbidden properties street, streetNumber, zip, city, country!
discovergy.0	2021-03-08 21:59:45.508	error	(69134) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
discovergy.0	2021-03-08 21:59:45.506	info	(69134) Discovergy meter found at your account with serial : XXXXXXXXX
discovergy.0	2021-03-08 21:59:45.504	error	(69134) State type : meteringLocationNumber unknown, send this information to the developer ==> meteringLocationNumber : "DE0XXXXXXXXXXXXXXXXXXXX"
discovergy.0	2021-03-08 21:59:45.371	info	(69134) Discovergy Adapter startet, trying to discover meters associated with your account
discovergy.0	2021-03-08 21:59:45.349	info

**Comment by DutchmanNL** [comment]
Which version did you try ? I cannot reproduce it with the semi account and version 0.5.4

**Comment by barni** [comment]
Hi,
it strange. I am a new ioBrooker user. Installed version seems to be 0.5.0. In ioBroker UI i cann only update to 0.5.0
Verfügbare Version:0.5.0
Installierte Version:0.5.0
In Git hub I can see Tags fpr 0.5.2.

I noticed the change also in my own JavaCode (Own Discovergy cod to fill an influx db) I had to add meteringLocationNumber. But sadly I am not able to debug JavaScript because I dont have JavaScript knowledge.

Cheers,
Stephan
